docs/sources/setup/install/helm/deployment-guides/azure.md

```diff
@@ -0,0 +1,588 @@
+    --resource-group <MY_RESOURCE_GROUP_NAME>
+    ```
+    **Replace the placeholders with your desired values.** We assume you have a resource group created.
+
+1. Create the containers for chunks and ruler:
+
+    ```bash
+    az storage container create --account-name <STORAGE-ACCOUNT-NAME> --name <CHUNK-BUCKET-NAME> --auth-mode login && \
+    az storage container create --account-name <STORAGE-ACCOUNT-NAME> --name <RULER-BUCKET-NAME> --auth-mode login
+    ```
+    **Make sure `--account-name` matches the account you just created**
+
+
+With the storage account and containers created, you can now proceed to creating the Azure AD role and federated credentials.
+
+## Creating the Azure AD Role and Federated Credentials
+
+The recommended way to authenticate Loki with Azure Blob Storage is to use federated credentials. This method is more secure than hard coding a connection string directly into the Loki configuration. In this next section, we will create an Azure AD role and federated credentials for Loki to allow it to read and write from Azure Blob Storage:
+
+1. Locate the OIDC issuer URL:
+
+    ```bash
+    az aks show \
+    --resource-group <MY_RESOURCE_GROUP_NAME> \
+    --name <MY_AKS_CLUSTER_NAME> \
+    --query "oidcIssuerProfile.issuerUrl" \
+    -o tsv
+    ```
+    This command will return the OIDC issuer URL. You will need this URL to create the federated credentials.
+
+1. Generate a `credentials.json` file with the following content:
+    ```json
+    {
+        "name": "LokiFederatedIdentity",
+        "issuer": "<OIDC-ISSUER-URL>",
+        "subject": "system:serviceaccount:loki:loki",
+        "description": "Federated identity for Loki accessing Azure resources",
+        "audiences": [
+          "api://AzureADTokenExchange"
+        ]
+    }
+    ```
+    **Replace `<OIDC-ISSUER-URL>` with the OIDC issuer URL you found in the previous step.** Save this file for later use.
+
+1. Next generate a azure directory `app` we will use this to assign our federated credentials to:
+   ```bash
+    az ad app create \
+    --display-name loki \
+    --query appId \
+    -o tsv
+   ```
+    This will return the app ID. Save this for later use. if you need to find the app ID later you can run the following command:
+    ```bash
+    az ad app list --display-name loki --query "[].appId" -o tsv
+    ```
+
+1. The app requires a service principal to authenticate with Azure AD. Create a service principal for the app:
+
+    ```bash
+    az ad sp create --id <APP-ID>
+    ```
+    **Replace `<APP-ID>` with the app ID you generated in the previous step.**
+
+1. Next assign the federated credentials to the app:
+
+    ```bash
+    az ad app federated-credential create \
+      --id <APP-ID> \
+      --parameters credentials.json 
+    ```
+    Replace `<APP-ID>` with the app ID you generated in the previous step. **Make sure you have also updated and saved the `credentials.json` file before running this step.**
```

Comment:
**Make sure you have also updated and saved the `credentials.json` file before running this step.**
Maybe make this another step, BEFORE this one?